proxy/proxy_test.go

```diff
@@ -276,3 +276,73 @@ func TestNewReverseProxy(t *testing.T) {
 		t.Errorf("got body %q; expected %q", g, e)
 	}
 }
+
+func TestRouteMatcherFuncFromPolicy(t *testing.T) {
```

Comment:
😍

Reply:
Ahh, maybe add a test for trailing slash on a naked host?

Reply:
I added a test with a trailing slash and it worked.